package.json

```diff
@@ -1,16 +1,18 @@
 {
-  "name": "node-ext2fs",
+  "name": "ext2fs",
   "version": "0.0.2",
   "description": "NodeJS native bindings to the libext2fs for cross-platform ext filesystem handling",
   "author": "Petros Angelatos <petrosagg@resin.io>",
   "license": "Apache-2.0",
   "main": "index.js",
+  "gypfile": true,
```

Comment:
It's not necessary, but [mentioned](https://github.com/nodejs/node-addon-examples/tree/master/1_hello_world#step-3) that it should be added, so that npm knows it's a binary module without having to guess based on package contents.